### DISCUSSION TITLE
Can Bouncer be used to filter results of a query?

### DISCUSSION THREAD

**Discussion by guntribam** [adonisjs/core#3068] [Help] [answered]
I don't know if this is inside Bouncer scope.

I'm filtering the results of a query based on user access. If the user 'A' have the profile 'X' he can see 2 of 3 items of the list returned from lucid...

I know that you can filter this kind of thing in the query itself, but i'm imagining that Bouncer can filter the list that contains all items...

Can this kind of thing be filtered by Bouncer?

**Comment by thetutlage** [comment, marked as answer]
Nope, its outside the scope of bouncer. Either, you can use Model query scopes or use a package like this https://github.com/lookinlab/adonis-lucid-filter
